### ISSUE TITLE
Problem with this.storage in label.js

### ISSUE THREAD

**Issue by kifirkin** [SteelzZ/au-cmsinn#1]
I've get the following error when adding au-cmsinn to existing app:
 `Uncaught TypeError: Cannot read property 'update' of undefined label.js:89`
I think there is a lack of parameters here.
`CmsInnLabel = new LabelPlugin(gLabelName, gContentType, '['+gSelector+']'/*, new LabelPluginUI()*/);`

**Comment by SteelzZ** [comment]
Hi,
storage is set during configuration. I believe you are missing this bit in your project:

```
if (Meteor.isClient) {
    CmsInn.configure({
        layoutTemplate: 'layout'
    });
}
if (Meteor.isServer) {
    CmsInn.configure({
        plugins: {
            versioning:{
                insecure : true
            }
        }
    }); 
}
```

I have updated README and included this. Also check this [blog post](http://codetemple.blogspot.de/2014/08/awareunit-cmsinn-tutorial-one-about.html) I've tried to explain in details how to integrate this package in fresh project.

Let me know if this won't fix the problem and thanks for reporting this ;)
